core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -931,6 +937,8 @@ private MutationSet sendMutationsToTabletServer(String location,
 
       timeoutTracker.startingWrite();
 
+      Long usid = null;
```

Comment:
OptionalLong is a better style here, as it's more explicit than null checking and implicit reliance on auto-boxing.

Reply:
changed in a513d75.  Didn't see much harm or benefit in this change after making it.